auth/src/main/java/com/firebase/ui/auth/ui/email/CheckEmailFragment.java

```diff
@@ -132,6 +134,12 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
     }
 
     @Override
+    public void onSaveInstanceState(Bundle outState) {
+        outState.putBoolean(ExtraConstants.HAS_EXISTING_INSTANCE, true);
```

Comment:
Because Firebase now includes screen analytics, this is kinda pointless. (Analytics saves stuff in the bundle so it's never null on configuration change)

@samtstern Do you think we should keep this or just check for a null bundle in onCreate assuming that it will not be null because of Firebase Analytics?